qutebrowser/test/mainwindow/conftest.py

```diff
@@ -0,0 +1,34 @@
+"""
+pytest fixtures and utilities for testing.
+
+Fixtures defined here will be visible to all test files in this directory and
```

Comment:
What makes this file special is just its name: `conftest.py` files are loaded by pytest automatically and their fixtures are made available to all test files below or in the same directory as the `conftest.py` file. [Reference](http://pytest.org/latest/plugins.html#conftest-py-local-per-directory-plugins).